concepts/05 Widgets/Form/01 Getting Started with Form/70 Validate the Form Data.md

```diff
@@ -0,0 +1,224 @@
+DevExtreme includes a validation engine that checks edited values before they are saved. This engine supports different validation rule types, such as [Email](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/EmailRule/), [Numeric](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/NumericRule/), [Pattern](/Documentation/ApiReference/UI_Widgets/dxValidator/Validation_Rules/PatternRule/), and more. 
```

Comment:
>before they are saved

"saved"? wtf?
processed/submitted/handled?

There are 4 places when values are checked in form: initial show + editor value change + "button{ useSubmitBehavior: true} " + form.validate()

Reply:
Hmm, when we change an editor's value and then move the focus out, the validation triggers. How can we call this moment?